org.kde.kst.yml

```
@@ -0,0 +1,99 @@
+app-id: org.kde.kst
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: kst2
+finish-args:
+  - '--share=network'
+  - '--share=ipc'
+  - '--socket=x11'
```

does it support wayland? if yes:
```suggestion
  - '--socket=fallback-x11'
```

otherwise remove the wayland socket.

---

Fixed